content/reference/plugins/teamcity/index.md

```diff
@@ -53,12 +55,10 @@ Add a new build feature called **XML report processing**. Choose **Ant JUnit** a
 
 ## Usage
 
-A new Gatling FrontLine simulation will be started every time the job is run. Check the Console Log to check the advancement of the simulation. If the simulation ran successfully, it will looks like the following:
+A new Gatling Enterprise simulation will be started every time the job is run. Check the Console Log to check the advancement of the simulation. If the simulation ran successfully, it will looks like the following:
```

Comment:
suggestion```
A new Gatling Enterprise simulation will be started every time the job is run. Check the Console Log to see the advancement of the simulation. If the simulation ran successfully, it will look like the following:
```